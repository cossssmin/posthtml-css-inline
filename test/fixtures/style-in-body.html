<!DOCTYPE html>
<html>
<body>
  <style>
    .text-sm {
      font-size: 12px;
    }

    .text-blue {
      color: blue;
    }

    div {
      display: flex;
    }

    div {
      font-size: 14px !important;
    }
  </style>
  <div class="text-blue text-sm">text-sm</div>
</body>
</html>
