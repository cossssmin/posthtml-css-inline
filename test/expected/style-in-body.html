<!DOCTYPE html>
<html>
<body>
  <style>
    .text-sm {
      font-size: 12px;
    }

    .text-blue {
      color: blue;
    }

    div {
      display: flex;
    }

    div {
      font-size: 14px !important;
    }
  </style>
  <div class="text-blue text-sm" style="font-size: 14px; color: blue; display: flex">text</div>
</body>
</html>
